pyro/primitives.py

```diff
@@ -273,9 +273,9 @@ def module(name, nn_module, update_module_params=False):
 
 
 def random_module(name, nn_module, prior, *args, **kwargs):
-    """
+    r"""
     Places a prior over the parameters of the module `nn_module`.
-    Returns a distribution (callable) over `nn.Module`s, which
+    Returns a distribution (callable) over `nn.Module`\s, which
```

Comment:
This fixes the format at http://docs.pyro.ai/en/0.3.0-release/primitives.html#pyro.random_module. I don't know a better solution.